src/main/kotlin/no/nav/syfo/application/IPdfRepository.kt

```diff
@@ -0,0 +1,15 @@
+package no.nav.syfo.application
+
+import no.nav.syfo.infrastructure.database.dialogmote.database.domain.PPdf
+import java.sql.Connection
+
+interface IPdfRepository {
+
+    suspend fun getPdf(id: Int): PPdf
+
+    suspend fun createPdf(
+        connection: Connection, // To be removed
```

Comment:
Ikke fan av kommentarer in the wild sånn som dette – tenkte bare det var greit med en kommentar som indikerer at dette ikke egentlig er hvordan vi ønsker å gjøre det 🤔